lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
```ruby
Coursemology::Polyglot::Language.config do |config|
  config.language Cousemology::Polyglot::Language::Python::Python3Point4
end
```
Something like this?

Yah, I know why it's a extension, but the code readability is compromised IMO.